### DISCUSSION TITLE
does weblate use obsoleted strings ?

### DISCUSSION THREAD

**Discussion by strk** [WeblateOrg/weblate#9982] [General]
We're experimenting with the [Update PO files to match POT (msgmerge)](https://weblate.osgeo.org/addon/599/) addon and noticed obsoleted strings are retained when the PO file is updated. I believe this is done by msgmerge which has no switch to turn it off. When we update the .po ourselves we additionally drop those obsoleted strings using msgattrib --no-obsolete because we have no use for them.

Does weblate have any use for them ? Is there any way to request NOT including them or removing them before pushing the PO changes back into the repository ?

**Comment by nijel** [comment]
Weblate does not use such strings right now, see https://github.com/WeblateOrg/weblate/issues/1993

**Comment by emmapeel2** [comment]
I would like to have an option to remove obsolete strings from the .po files on the "Update PO files to match POT (msgmerge)" add-on.

I do it with a script once in a while, as it locks projects when recovering a string that was used before, but removed.
